spec/lib/miq_expression_spec.rb

```diff
@@ -2510,6 +2510,15 @@
         expect(result.map(&:first)[0]).to eq(" CPU Total Cost")
       end
     end
+
+    context "with :include_id_columns" do
+      it "VmOrTemplate" do
+        result = described_class.model_details("VmOrTemplate", :include_id_columns => true)
+        expect(result.map(&:second).include?("Vm-id"))
```

Comment:
These tests are all false positives because no matcher has been passed to the expectation:

```
it "passes" do
  expect(false)
end

# ....

Randomized with seed 25262
.

Finished in 0.00324 seconds (files took 4.66 seconds to load)
1 example, 0 failures
```

Perhaps:

```ruby
expect(result).to match_array([anything, "Vm-id"], [anything, "Vm-host_id"], [anything, "Vm-host_id"])
```